python/cuml/dask/common/base.py

```diff
@@ -407,12 +407,13 @@ def _fit(self, model_func, data):
             ]
         )
 
+        fit_func = self._func_fit
```

Comment:
This seems mostly a stylistic change, or was there a particular reason that prompted this change?

Reply:
yeah, the change allows overriding of the _func_fit function.  This is required by class dask.linear_model.LogisticRegression(SyncFitMixinLinearModel), which overrides the function in order to reuse SyncFitMixinLinearModel.fit().